pkg/cloudprovider/cloud.go

```diff
@@ -51,6 +51,7 @@ type Clusters interface {
 	Master(clusterName string) (string, error)
 }
 
+// TODO(#43173): Deprecate this common GetLoadBalancerName() function and use per-provider getLoadBalancerName() instead.
```

Comment:
I would add a target date, like "after v1.9 ships", and even rename the function to `DeprecatedGetLoadBalancerName()`